AIFS_ROS/hiperlab_rostools/src/AirSimBridge/main.cpp

```diff
@@ -44,12 +44,22 @@
 #include "std_msgs/Bool.h"
 using namespace std;
 
-class ImagePollAgent {
+class SingleImagePollAgent {
  public:
+  SingleImagePollAgent(ImageCaptureBase::ImageType requestedImageType =
```

Comment:
Confused when reading the code. It is single image poll agent, but why inputting two types?